terraform/alerting/monitoring.tf

```diff
@@ -68,3 +68,72 @@ resource "google_logging_metric" "stackdriver_export_error_count" {
     value_type  = "INT64"
   }
 }
+
+resource "google_logging_metric" "human_accessed_secret" {
+  name    = "human_accessed_secret"
+  project = var.project
+
+  filter = <<EOT
+resource.type="audited_resource"
+resource.labels.service="secretmanager.googleapis.com"
+resource.labels.method:"AccessSecretVersion"
```

Comment:
You can merge these two metrics together with a filter on `resource.label.method`. If the goal is to set up alert regardless which metric is triggered, there's no need to have two metrics.

You can extract method as a label if needed.

Reply:
I want to keep them separate because some stakeholders only care about secrets. Once I figure out the alerts, "alert_on_kms" will be configurable, so I think it'll be easier to keep them separate